docs/developer/tutorials/snmp/profiles.md

```diff
@@ -79,6 +79,93 @@ metrics:
 !!! tip
     `sysobjectid` can also be a wildcard pattern to match a sub-tree of devices, eg `1.3.6.1.131.12.4.*`.
 
+### Generate a profile file from a collection of MIBs
+
+You can use `ddev` to create a profile from a list of mibs.
+
+```console
+$  ddev meta snmp generate-profile-from-mibs --help
+```
+
+This script requires a list of ASN1 MIB files as input argument, and copies to the clipboard a list of metrics that can be used to create a profile.
+
+#### Options
+
+`-f, --filters` is an option to provide the path to a YAML file containing a collection of MIB names and their list of node names to be included.
```

Comment:
@sarina-dd I changed I changed this description since your review, is it still ok?

Reply:
LGTM